src/app/shared/components/number-stepper/template.html

```diff
@@ -21,6 +21,7 @@
          matInput
          type="number"
          autocomplete="off"
+         onkeydown="return event.keyCode !== 69"
```

Comment:
We shouldn't be ignore `e` in this component since its a generic component but instead this should be handled in the parent component where we want to ignore `e`

Reply:
when we enter `e` in the text input it clears the input, this issue is present everywhere this component is being used.
This was done after discussion with @ahmedwaleedmalik